### DISCUSSION TITLE
[Enhancement] [cascader] el-cascader 的 before-filter 参数在将输入框的东西清空后没有重新调用给的方法

### DISCUSSION THREAD

**Discussion by dmskys** [element-plus/element-plus#18223] [Feature Request]
## Description

el-cascader 的 before-filter 参数在将输入框的东西清空后没有重新调用给的方法

**Comment by dmskys** [comment]
el-cascader 懒加载加远程搜索功能，需要在监听清空搜索框，发现before-filter 清空搜索字符没有触发事件导致无法恢复组件初始化状态

**Reply by warmthsea** [reply]
请提供一个最小复现，谢谢
